core-bundle/src/EventListener/DataContainer/DefaultOperationsListener.php

```diff
@@ -186,7 +186,7 @@ private function accessChildrenCallback(string $ctable, string $table): \Closure
     {
         return function (DataContainerOperation $operation) use ($ctable, $table): void {
             $data = [
-                'pid' => $operation->getRecord()['pid'] ?? null,
+                'pid' => $operation->getRecord()['id'] ?? null,
```

Comment:
Is this a related change?

Reply:
yes, that is why the current voter worked in the article list (children operation), but it is inconsistent with what DC_Table votes on in the parent view (tl_content).